frontend/packages/ceph-storage-plugin/src/components/ocs-install/node-list.tsx

```diff
@@ -147,13 +153,46 @@ const CustomNodeTable: React.FC<CustomNodeTableProps> = ({ data, loaded, ocsProp
   const [error, setError] = React.useState('');
   const [inProgress, setProgress] = React.useState(false);
   const [selectedNodesCnt, setSelectedNodesCnt] = React.useState(0);
+  const [hasInvalidNodes, setHasInvalidNodes] = React.useState(false);
+  const [nodesWarningMsg, setNodesWarningMsg] = React.useState('');
 
   let storageClass = '';
 
+  const validateCPU = (node: formattedNodeType) => {
+    return Number(_.get(node, 'cells[3].title')) < MIN_NODE_CPU;
+  };
+
+  const validateMemory = (node: formattedNodeType) => {
+    return convertToBaseValue(_.get(node, 'cells[4].title')) < MIN_NODE_MEMORY;
+  };
+
+  const validateNodes = React.useCallback((selectedNodes: formattedNodeType[]) => {
+    let invalidNodesCount = 0;
+    let nodeName = '';
+    selectedNodes.forEach((node) => {
+      if (validateCPU(node) || validateMemory(node)) {
+        invalidNodesCount++;
+        nodeName = node.id;
+      }
+    });
+
+    if (invalidNodesCount > 0) {
+      setHasInvalidNodes(true);
+      const msg =
+        invalidNodesCount > 1
+          ? `${invalidNodesCount} of the selected nodes does not meet minimum requirements of 16 CPU/ 64 GB Memory`
```

Comment:
Do you mean 16 cores? And it's GiB, not GB. Those values are different.

```suggestion
          ? `${invalidNodesCount} of the selected nodes do not meet minimum requirements of 16 cores and 64 GiB memory.`
```

Reply:
We're already hard-coding the values in the message, so I wouldn't create a constant in another file. That's just asking for these values to get out of sync. It's only used in one place. Better to inline the expected value in this file.

Reply:
@yuvalgalanti @shirimordechay ^

Reply:
@spadgett  This file is created only for ocs install flow constants. Hence, I have added these constants there.

Reply:
But you've also hard-coded the values into the messages, so you've effectively defined them in multiple places. It would be very easy to change the constant without changing the message later, causing a bug. In a way, the constant is working against you here. Since you only use the constant in one place, it would be better to simply inline the value.